src/components/stepforms/widgets/MultiInputText.vue

```diff
@@ -1,26 +1,38 @@
 <template>
   <div class="widget-multiinputtext__container">
-    <multiselect
+    <VariableInput
```

Comment:
Using a slot is really cool ! It would be cool in the future to "tag" the widget which use it. The goal would be to programatically separate widgets which can handle variable input from the others.

Reply:
But I do not know how we could do this for now 🤔